api/client.go

```diff
@@ -56,12 +55,8 @@ func New(ctx context.Context, apiClusterContext, project string, opts ...ClientO
 		return nil, err
 	}
 
-	mapper := apis.StaticRESTMapper(scheme)
```

Comment:
I have removed this since we now use the lazy mapper that comes with the new controller-runtime.